schema.graphql

```diff
@@ -99,6 +99,16 @@ input CommentInput {
   text: String!
 }
 
+type Community {
+  _id: ID!
+  createdAt: DateTime
+  description: String!
+  image: String
```

Comment:
Rename to 'logoUrl'

Reply:
A community means all the organizations managed by an API instance.